tests/test_browser.py

```diff
@@ -4985,6 +4985,18 @@ def test_embind_with_pthreads(self):
   def test_embind_with_asyncify(self):
     self.btest('embind_with_asyncify.cpp', '1', args=['--bind', '-s', 'ASYNCIFY'])
 
+  @parameterized({
+    '': ([],),
+    'no_dynamic_execution': (['-s', 'DYNAMIC_EXECUTION=0'],)
+  })
+  def test_embind_lib_with_asyncify(self, args):
+    self.btest('embind_lib_with_asyncify.cpp', 'ok', args=[
+      '--bind',
+      '-s', 'ASYNCIFY',
+      '-s', 'ASYNCIFY_IMPORTS=["sleep_and_return"]',
+      '--post-js', test_file('embind_lib_with_asyncify.test.js'),
+    ] + args)
```

Comment:
It would be better to not do this as a browser test, I think - we can test async things in `test_core.py`. See `test_async_ccall*` there.

Reply:
(browser tests are much slower)

Reply:
Oh yeah good point. I just put it alongaide the other embind+asyncify test, but I think it was added by me here as well and should've been in core instead also. I'll fix tomorrow.

Reply:
Ah I probably put the previous test here because there was `test_embind_with_pthreads` above. Should we move it as well?

Reply:
I see, `test_embind_with_asyncify` uses `fetch` and probably should remain a browser test. Let's see if I can move at least this one...

Reply:
Yeah moved this test just fine.